core/lib/tasks/solidus/import_existing_permission_sets.rake

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+namespace :solidus do
+  desc "Import existing permission sets to role permissions table"
+  task import_existing_permission_sets: :environment do
+    Spree::PermissionSets::Base.subclasses.each do |permission|
```

Comment:
Should we prepend with `Zeitwerk::Loader.eager_load_all`? Otherwise, we might skip subclasses that haven't yet been loaded in the development environment.

Also, should we consider subclasses of subclasses?

For both questions, I'm thinking about user-defined permission sets, as they're not a problem in vanilla Solidus.

Reply:
Great suggestion! 

Did some tests with custom permission sets on Sandbox and it worked.
Also changed from `subclasses` to `descendants`, so we do get subclasses of subclasses as well as permission sets inherited from other permission sets.